src/settings-ui/Settings.UI/OOBE/Views/OobeShellPage.xaml.cs

```diff
@@ -49,6 +50,9 @@ public static void SetOpenMainWindowCallback(Action<Type> implementation)
 
         public OobeShellPage()
         {
+            Experiments exp = new Experiments();
+            experimentEnabled = exp.EnableLandingPageExperiment();
```

Comment:
There's only one experiment for now, but a more descriptive name that indicates which experiment this is still seems helpful.